### ISSUE TITLE
SCSS deprecation warnings

### ISSUE THREAD

**Issue by Spone** [wearefine/fae#303]
This is apparently related to https://github.com/sass/sass/issues/1599 (it's no longer allowed to extend a compound selector, ie. `.icon-something:before`).

I use sass 3.5.1 and sass-rails 5.0.6

```
11:16:27 web.1       | DEPRECATION WARNING on line 10 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/_plugins.scss:
11:16:27 web.1       | Extending a compound selector, .icon-arrow_down:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 29 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/_plugins.scss:
11:16:27 web.1       | Extending a compound selector, .icon-arrow_up:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 36 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/_plugins.scss:
11:16:27 web.1       | Extending a compound selector, .icon-search:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 10 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/layout/_breadcrumbs.scss:
11:16:27 web.1       | Extending a compound selector, .icon-chevron_right:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 70 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/navigation/_sidenav.scss:
11:16:27 web.1       | Extending a compound selector, .icon-arrow_down:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 120 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/navigation/_sidenav.scss:
11:16:27 web.1       | Extending a compound selector, .icon-arrow_up:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 31 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/navigation/_mobilenav.scss:
11:16:27 web.1       | Extending a compound selector, .icon-chevron_right:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 59 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/navigation/_mobilenav.scss:
11:16:27 web.1       | Extending a compound selector, .icon-chevron_left:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 118 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/navigation/_mobilenav.scss:
11:16:27 web.1       | Extending a compound selector, .icon-logout:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 124 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/navigation/_mobilenav.scss:
11:16:27 web.1       | Extending a compound selector, .icon-settings:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 130 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/navigation/_mobilenav.scss:
11:16:27 web.1       | Extending a compound selector, .icon-support:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 136 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/navigation/_mobilenav.scss:
11:16:27 web.1       | Extending a compound selector, .icon-offlink:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 81 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/navigation/_utility.scss:
11:16:27 web.1       | Extending a compound selector, .icon-arrow_down:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 42 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/tables/_sorting.scss:
11:16:27 web.1       | Extending a compound selector, .icon-arrow_down:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 56 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/tables/_sorting.scss:
11:16:27 web.1       | Extending a compound selector, .icon-arrow_up:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 9 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/tables/_collapsible.scss:
11:16:27 web.1       | Extending a compound selector, .icon-arrow_up:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 31 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/tables/_collapsible.scss:
11:16:27 web.1       | Extending a compound selector, .icon-arrow_down:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 55 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/tables/_collapsible.scss:
11:16:27 web.1       | Extending a compound selector, .icon-arrow_down:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 73 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/tables/_collapsible.scss:
11:16:27 web.1       | Extending a compound selector, .icon-arrow_up:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 32 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/_buttons.scss:
11:16:27 web.1       | Extending a compound selector, .icon-heavy_plus:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 69 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/_modal.scss:
11:16:27 web.1       | Extending a compound selector, .icon-delete_x:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 27 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/_alerts.scss:
11:16:27 web.1       | Extending a compound selector, .icon-check:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 37 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/_alerts.scss:
11:16:27 web.1       | Extending a compound selector, .icon-alert:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 47 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/_alerts.scss:
11:16:27 web.1       | Extending a compound selector, .icon-alert:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 86 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/forms/_base.scss:
11:16:27 web.1       | Extending a compound selector, .icon-plus:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 13 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/forms/_checkbox.scss:
11:16:27 web.1       | Extending a compound selector, .icon-check:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 66 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/forms/_asset-actions.scss:
11:16:27 web.1       | Extending a compound selector, .icon-trash:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 105 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/forms/_select.scss:
11:16:27 web.1       | Extending a compound selector, .icon-delete_x:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 278 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/forms/_select.scss:
11:16:27 web.1       | Extending a compound selector, .icon-delete_x:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 284 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/forms/_select.scss:
11:16:27 web.1       | Extending a compound selector, .icon-check:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 131 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/forms/_date.scss:
11:16:27 web.1       | Extending a compound selector, .icon-chevron_left:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 139 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/forms/_date.scss:
11:16:27 web.1       | Extending a compound selector, .icon-chevron_right:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 202 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/forms/_date.scss:
11:16:27 web.1       | Extending a compound selector, .icon-calendar:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 269 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/forms/_date.scss:
11:16:27 web.1       | Extending a compound selector, .icon-chevron_right:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 276 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/forms/_date.scss:
11:16:27 web.1       | Extending a compound selector, .icon-chevron_left:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 9 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/modules/forms/_hints.scss:
11:16:27 web.1       | Extending a compound selector, .icon-support:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 457 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/legacy/_pre-1.3.scss:
11:16:27 web.1       | Extending a compound selector, button[type=button], is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 463 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/legacy/_pre-1.3.scss:
11:16:27 web.1       | Extending a compound selector, .icon-arrow_down:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 537 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/legacy/_pre-1.3.scss:
11:16:27 web.1       | Extending a compound selector, .icon-alert:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
11:16:27 web.1       | 
11:16:27 web.1       | DEPRECATION WARNING on line 600 of /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/assets/stylesheets/fae/globals/legacy/_pre-1.3.scss:
11:16:27 web.1       | Extending a compound selector, .icon-delete_x:before, is deprecated and will not be supported in a future release.
11:16:27 web.1       | See https://github.com/sass/sass/issues/1599 for details.
```

**Comment by jamesmk** [comment]
v2.0 will be the next dependency version updates. We'll need to resolve this for that release for sure. cc @aaronransley

**Comment by tshedor** [comment]
@jamesmk this was fixed in #349 and can be closed
